### PR TITLE
Github add Issue / PR Templates

### PR DESCRIPTION
**Pull request by gomezzz**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Description
+
+Summary of changes
+
+* ... 
+* ...
+## Resolved Issues
```
Comment (htoftevaag) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: Should we have whitespace before?
```suggestion

## Resolved Issues
```

Comment (htoftevaag) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Also, is there a reason why some of the bullet points have * and others -?

**Review by htoftevaag — CHANGES_REQUESTED** [review]
Two minor changes

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (gomezzz) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: They are interchangeable

**Pull request merged**